src/Optimizer/AmpWPConfiguration.php

```diff
@@ -66,6 +66,7 @@ public function apply_filters() {
 		}
 
 		array_unshift( $transformers, WpTransformer\AmpSchemaOrgMetadata::class );
+		array_unshift( $transformers, WpTransformer\DetermineHeroImages::class );
```

Comment:
Or alternatively:

```suggestion
		array_unshift(
			$transformers,
			WpTransformer\DetermineHeroImages::class,
			WpTransformer\AmpSchemaOrgMetadata::class
		);
```

I'd like to think doing one `array_unshift()` call is more performant than doing several.